query-engine/request-handlers/src/load_executor.rs

```diff
@@ -24,6 +24,9 @@ pub async fn load(
         #[cfg(feature = "mongodb")]
         p if MONGODB.is_provider(p) => mongodb(source, url, features).await,
 
+        #[cfg(feature = "js-connectors")]
+        p if JsConnector::is_provider(p) => jsconnector(source, url, features).await,
+
```

Comment:
In the future, it might be interesting to **not** compile the rest of the modules corresponding to connectors not used. This would be all connectors for sql flavors that are not supported in serverless, although eventually we might have serverless connectors for all 6 flavors of SQL we support: Mongo, MSSQL, mysql, postgres, CDB, sqlite.

Reply:
Hard agree on this one, as previously hinted at in our conversations